Functions/xASL_bids_BIDSifyASLNII.m

```diff
@@ -29,33 +28,44 @@
 if ~exist(pathTSV,'dir')
 	mkdir(pathTSV);
 end
+
+% Write volume types (asl context) to file
```

Comment:
Agree. Minor testing has to be done then.

Reply:
Yes, we even wrote that down in some other issue... I think. There are some places in the code where we should use `xASL_tsvWrite` and `xASL_csvWrite` instead of manually writing text to files. Just didn't find the time to do this so far 😄 More important for me would be to do this correctly where we write the csv import summary files. Those are written a **_wild_** way.

Reply:
This is something that doesn't have to be done in this PR though, right?

Reply:
Shall we make a separate issue for changing `fopen-sprintf-fclose` by `xASL_(csv|tsv)Write`?

Reply:
#901